packages/circuit-ui/components/SidePanel/SidePanelContext.spec.tsx

```diff
@@ -45,7 +45,7 @@ import {
   type SidePanelContextProps,
 } from './SidePanelContext.js';
 
-vi.mock('../../hooks/useMedia');
+vi.mock('../../hooks/useMedia/index.js');
```

Comment:
why so explicit ?

Reply:
The final ES module spec requires full file paths ([ref](https://nodejs.org/api/esm.html#mandatory-file-extensions)). I'm surprised that this worked before; I assume Vite does some magic under the hood to support ambiguous file paths.